<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>template_manage</class>
 <widget class="QDialog" name="template_manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>template_manage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>group manage:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="7">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="6" rowspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">.QPushButton {
	border-style: none;
	border: 0px;
	min-height:32px;
    min-width:70px;
	border-radius: 8px;
	color: rgb(255, 255, 255);
	
	background-color: rgb(26, 188, 156);
    background-repeat:no-repeat;
}


.QPushButton:hover{ 
background-color: rgb(74, 215, 187);
    background-repeat:no-repeat;
}

.QPushButton:pressed{ 
background-color: rgb(19, 159, 131);
}
</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>log type:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>new name:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="7">
    <widget class="QListWidget" name="listWidget">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="positionBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>localization</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="repositionBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>deviation</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QCheckBox" name="systemLogBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>system_log</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>422</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="3" colspan="2">
    <widget class="QPushButton" name="connectButton">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">.QPushButton {
	border-style: none;
	border: 0px;
	min-height:32px;
    min-width:70px;
	border-radius: 8px;
	color: rgb(255, 255, 255);
	
	background-color: rgb(26, 188, 156);
    background-repeat:no-repeat;
}


.QPushButton:hover{ 
background-color: rgb(74, 215, 187);
    background-repeat:no-repeat;
}

.QPushButton:pressed{ 
background-color: rgb(19, 159, 131);
}
</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5" colspan="2">
    <widget class="QPushButton" name="connectButton_2">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">.QPushButton {
	border-style: none;
	border: 0px;
	min-height:32px;
    min-width:70px;
	border-radius: 8px;
	color: rgb(255, 255, 255);
	
	background-color: rgb(26, 137, 188);
    background-repeat:no-repeat;
}


.QPushButton:hover{ 
background-color: rgb(88, 174, 214);
    background-repeat:no-repeat;
}

.QPushButton:pressed{ 
background-color: rgb(20, 113, 157);
}
</string>
     </property>
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>template_manage</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectButton_2</sender>
   <signal>clicked()</signal>
   <receiver>template_manage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
